staging/src/k8s.io/apiserver/pkg/storage/etcd3/store.go

```diff
@@ -107,8 +120,11 @@ func newStore(c *clientv3.Client, quorumRead, pagingEnabled bool, codec runtime.
 		// for compatibility with etcd2 impl.
 		// no-op for default prefix of '/registry'.
 		// keeps compatibility with etcd2 impl for custom prefixes that don't start with '/'
-		pathPrefix: path.Join("/", prefix),
-		watcher:    newWatcher(c, codec, versioner, transformer),
+		pathPrefix:              path.Join("/", prefix),
+		watcher:                 newWatcher(c, codec, versioner, transformer),
+		leaseLock:               sync.Mutex{},
+		prevLeaseExpirationTime: 0,
```

Comment:
These two lines are setting items to their zero values and can be omitted. Possibly the next, too. FYI